rasa/core/interpreter.py

```diff
@@ -296,6 +301,143 @@ def _load_interpreter(self) -> None:
 
         self.interpreter = Interpreter.load(self.model_directory)
 
+MODEL_NAME_PREFIX = "nlu"
+
+def find_same(action_name, attribute, training_data):
+        for ex in training_data.training_examples:
+            if ex.as_dict()['text'] == action_name and ex.get(SPARSE_FEATURE_NAMES[attribute]) is not None:
+                example = ex
+                break
+        return example
+
+class RasaCoreInterpreter(NaturalLanguageInterpreter):
+    def __init__(self,):
+        self._load_interpreter()
+
+    def parse(self, text: Text):
+        result = self.interpreter.parse(text, only_output_properties=False)
+        return result
+
+    def _load_interpreter(self) -> None:
+        from rasa.nlu import config
+
+        config = config.load()
+        self.trainer = Trainer(config)
+        self.interpreter = Interpreter(self.trainer.pipeline, {})
```

Comment:
I'd say it is `None` in init, because there is no `interpreter` until we train the `trainer`
```suggestion
        self.interpreter = None
```